app_integrations/apps/app_base.py

```diff
@@ -65,7 +68,7 @@ def get_app(config, init=True):
 class AppIntegration(object):
     """Base class for all app integrations to be implemented for various services"""
     __metaclass__ = ABCMeta
-    _POLL_BUFFER_MULTIPLIER = 1.2
+    _POLL_BUFFER_MULTIPLIER = 1.5
```

Comment:
I'd add a comment explaining what this is for

Reply:
This is explained when it's accessed here (where the context matters more than when it's defined): https://github.com/airbnb/streamalert/blob/3cf8b22dbc6b42a0b10db96a6a66d346c98995dc/app_integrations/apps/app_base.py#L310-L312

I  can add an extra comment here to be doubly sure it's seen